R/plotPercentiles.R

```diff
@@ -0,0 +1,107 @@
+#' Comparison plots show user selected variables obtained from different scenario
+#' runs
+
+#' @author Tonn Rueter
+#' @param df `quitte` style data frame containing all variables for each scenario
+#' @param scenarios Character vector contains names of the desired scenarios. If none, all scenarios will be displayed
+#' @param variables Character vector contains names of the desired variables. If none, all variables will be displayed
+#' @importFrom dplyr filter mutate vars
+#' @importFrom reshape2 melt
+#' @importFrom stringr str_extract
+#' @importFrom tidyr pivot_wider
+#' @importFrom ggplot2 ggplot geom_line geom_ribbon facet_wrap facet_grid theme ylab
+#' @export
+plotPercentiles <- function(df, scenarios = NULL, variables = NULL) {
+
+  # In the quitte data frame all perenctiles are given as individual variables
+  # Manipulate input data frame such that all percentiles of a given quantity
+  # are transformed to individual columns. Variable names in the quitte data
```

Comment:
Assumptions about the structure of the data frame should be moved to the description of the parameter `df` in the function documentation so they can be viewed when calling `help("plotIterations")`. User's should not have to look into the code to find out what the structure of the input data is.